app/lib/task/backend.dart

```diff
@@ -971,8 +972,9 @@ List<Version> _versionsToTrack(
     // Consider 5 latest major versions, if any:
     ...packageVersions
         // Ignore prereleases and retracted versions
-        .where((pv) => !pv.isRetracted && !pv.semanticVersion.isPreRelease)
-        .map((pv) => pv.semanticVersion)
+        .where((pv) => !(pv.retracted ?? false))
+        .map((pv) => Version.parse(pv.version))
+        .where((v) => !v.isPreRelease)
```

Comment:
I'd love this fix, can we split it out? But curious why not use `pv.isRetracted`?

Reply:
Yeah, I'll do a split out PR.